Eatery Watch App Extension/Notification (Unused)/NotificationController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NotificationController.swift
+//  Eatery Watch App Extension
+//
+//  Created by William Ma on 1/5/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import WatchKit
+import SwiftUI
+import UserNotifications
+
+class NotificationController: WKUserNotificationHostingController<NotificationView> {
+
+    override var body: NotificationView {
+        return NotificationView()
+    }
+
+    override func willActivate() {
+        // This method is called when watch view controller is about to be visible to user
+        super.willActivate()
+    }
+
+    override func didDeactivate() {
+        // This method is called when watch view controller is no longer visible
+        super.didDeactivate()
+    }
+
+    override func didReceive(_ notification: UNNotification) {
+        // This method is called when a notification needs to be presented.
+        // Implement it if you use a dynamic notification interface.
+        // Populate your dynamic notification interface as quickly as possible.
+    }
```

Comment:
are these necessary? you dont seem to add any additional logic to them

Reply:
I'm not sure what these are used for, but they were added to the app and I was scared to remove them. I've taken them out and tested the app lightly again.